src/libs/ajax/DataRepo.ts

```diff
@@ -119,11 +121,17 @@ const callDataRepo = async (url: string, signal?: AbortSignal) => {
   return await res.json();
 };
 
+const callDataRepoPost = async (url: string, signal: AbortSignal | undefined, jsonBodyArg?: object): Promise<any> => {
+  return await fetchDataRepo(url, _.mergeAll([authOpts(), jsonBody(jsonBodyArg), { signal, method: 'POST' }]));
```

Comment:
Does `jsonBody()` work if its arg is undefined? Maybe you want something like
```suggestion
  return await fetchDataRepo(url, _.mergeAll([authOpts(), jsonBodyArg && jsonBody(jsonBodyArg), { signal, method: 'POST' }]));
```